linux/shim/init.go

```diff
@@ -237,31 +237,30 @@ func (p *initProcess) SetExited(status int) {
 }
 
 func (p *initProcess) Delete(context context.Context) error {
-	status, err := p.Status(context)
-	if err != nil {
-		return err
-	}
-	if status != "stopped" {
```

Comment:
The client just need to call `KillAll` before hands, which it was it does now. Anyhow, since `runc` seems to mess with the state, doesn't seem like there's another choice than not checking it here.